src/main/java/org/coffeemine/app/spring/UserProfile.java

```diff
@@ -0,0 +1,18 @@
+package org.coffeemine.app.spring;
+
+import com.vaadin.flow.router.Route;
+
+@Route
+public class UserProfile extends View {
+
+    UserProfile() {
+    super();
+    add(new profileView());
```

Comment:
This file is unneeded, profile view is separate and should not be part of the View.
Instead profileView should be accessed via the account button in the future, but this is not required by the current issue.

Reply:
ah, I will remove this

Reply:
Please remove the old file :)